integration/Substra/assets/dataset/opener_test_MyOrg1MSP.py

```diff
@@ -0,0 +1,24 @@
+from flamby.datasets.fed_tcga_brca import FedTcgaBrca
```

Comment:
@Esadruhn @ThibaultFy would it be possible to add class inheritance ? Aka can we pass a base class to the openers's dockerfile somehow ? This would allow to factorize all openers real easily.

Reply:
I am wondering also if we can do that if it wouldn't be better to find a way to write only one opener and to dynamically send the appropriate instance of it when running the script (add the center number only or train/test) but maybe it's too much and ends up being even more confusing than having them in clear. What do you think @Esadruhn @ThibaultFy ?

Reply:
An other way to do it is to have config json files that we load as datasamples instead of giving empty paths, and to load the config into a single opener. 
I found that bypassing the real behaviour of datasamples to pass config files was not the spirit. we are looking for in a documentation example. But I also agree that it'll remove some complexity to the example...

Reply:
I think that one opener per center is in the spirit of Substra, with the paradigm that we have specific loaders per centers, and a common algorithm.
As you say, we could dynamically write the opener from the code, I am afraid this would bring much complexity for the end user